### PR TITLE
Add basic building instructions to README

### PR DESCRIPTION
**Pull request by jeid64**
Added instructions to build APK for developers. Included getting submodules and installing NDK.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,12 @@ This project is being actively developed at [XDA Developers](http://forum.xda-de
 1. Fork us
 2. Write code
 3. Send Pull Requests
+Building
```
Comment (cgutman) [inline README.md]: This line looks like it was added by mistake

File: README.md [hunk 2]
```diff
@@ -45,6 +45,12 @@ This project is being actively developed at [XDA Developers](http://forum.xda-de
...
+
+##Buidling
```
Comment (cgutman) [inline README.md]: Typo here

File: README.md [hunk 3]
```diff
@@ -45,6 +45,12 @@ This project is being actively developed at [XDA Developers](http://forum.xda-de
...
+* Install Android Studio and the Android NDK
+* Run git submodule update --init --recursive from within moonlight-android/
+* Build the APK using Android Studio
```
Comment (cgutman) [inline README.md]: We probably want to add the step of adding ndk.dir to the local.properties file

File: README.md [hunk 4]
```diff
@@ -45,6 +45,12 @@ This project is being actively developed at [XDA Developers](http://forum.xda-de
 1. Fork us
 2. Write code
 3. Send Pull Requests
+Building
+
+##Buidling
+* Install Android Studio and the Android NDK
+* Run git submodule update --init --recursive from within moonlight-android/
```
Comment (cgutman) [inline README.md]: I'd prefer to have the command in quotes like this:
* Run 'git submodule update --init --recursive' from the root of your moonlight-android repo

**Comment by jeid64** [comment]
@cgutman Thanks for looking this over! Sorry for the mistakes, must have not had my coffee when I wrote it.

**Pull request merged**